row_cache.cc

```diff
@@ -978,7 +983,6 @@ future<> row_cache::do_update(external_updater eu, replica::memtable& m, Updater
                             if (update.run() == stop_iteration::no) {
                                 return;
                             }
-                            update = {};
```

Comment:
This also delays freeing memory. I guess it's not important, we'll either replace `update` or end the loop soon.

Reply:
Right, I'll bring it back, just in case.